src/app/app.component.html

```diff
@@ -1,12 +1,14 @@
 <!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
-  <nav mat-tab-nav-bar>
-    <a class="nav-fill-space"></a>
-    <a mat-tab-link routerLink="/feeds" routerLinkActive="active">Feeds</a>
-    <a mat-tab-link routerLink="/downloads" routerLinkActive="active">Downloads</a>
-    <a mat-tab-link routerLink="/requests">Requests</a>
-    <a class="nav-fill-space"></a>
-  </nav>
-<!--  <app-feedtable></app-feedtable>-->
-  <router-outlet></router-outlet>
+    <nav mat-tab-nav-bar>
+        <a class="nav-fill-space"></a>
+        <a mat-tab-link routerLink="/feeds" routerLinkActive="active">Feeds</a>
+        <a mat-tab-link routerLink="/downloads" routerLinkActive="active"
+            >Downloads</a
```

Comment:
Some weird new line formatting going on here... intentional?

Reply:
Default prettier rules. This is intended and will be standardized across all UI for consistency.

Reply:
https://prettier.io/

Reply:
So `>Downloads</a` is an acceptable styling? TBH, it's really hard to read.

Reply:
Updated prettier rules. It should be easier on the eyes.